system_test/ResetTest.words

```diff
@@ -0,0 +1,12 @@
+Alex is a thing at 0,0.
+Alex's name is "alex".
+Make Alex say Alex's name.
+
+Reset.
+
+Alex is a thing at 0,0.
+Make Alex say Alex's name.
```

Comment:
This line will generate a runtime error, since right now an object can't say something that is NOTHING, and Alex's name is nothing at this point.  When I run the system_tests, the syntax error is printed, which is confusing.  Right now we should probably have all our system_test programs run without errors, and we'll figure out separately how to write tests that runtime errors are generated when we expect them to be.